resources/styles/old/global/buttons.less

```diff
@@ -7,18 +7,44 @@
 
 // Override the defaults from bootstrap-material-design
 
+.btn-round() {
+  margin-right: 5px;
+  padding: 9px 18px;
+  font-size: 1.6rem;
+  border-radius: @btn-border-radius;
+  transition: box-shadow 0.2s, background 0.2s;
+
+  &:hover {
+    box-shadow: 0 4px 10px rgba(0, 0, 0, .1), 0 6px 28px rgba(0, 0, 0, .2);
+  }
+}
+
+.btn-tutor-default() {
+  color: #424242 !important; //!important to overide local styles in plan-details.less. we can remove after integration
```

Comment:
Is the CSS in `plan-details` not scoped to be in a `plan`? And any reason this color isn't with the other colors in `variables.less`?